CSS/css_completions.py

```diff
@@ -164,7 +182,13 @@ def complete_property_value(self, view, prefix, pt):
 
         return completions
 
-    def complete_function_argument(self, view: sublime.View, prefix, pt):
+    def complete_function_argument(
+        self,
+        view: sublime.View,
+        prefix: str,
+        pt: sublime_types.Point
+    ) -> typing.Optional[typing.List[sublime.CompletionItem]]:
```

Comment:
I think you should switch to importing specific types from `typing` in the import statement because:
a) it makes code shorter, cleaner and easier to read
b) that seems to be the convention used in ST types.

And then there would also be even less reason to break the lines for each argument which takes a lot of vertical space.